### DISCUSSION TITLE
French language models

### DISCUSSION THREAD

**Discussion by Keeermalec** [oobabooga/text-generation-webui#4278] [Q&A]
looking for french language models for Oobabooga's text-generation-webui

The French language LLMs I download from Huggingface mostly dont work on this awesome GUI.

Any suggestions?

**Comment by flnh** [comment]
Hello @Keeermalec,

I have personally used : [Vigogne-2-7B-Instruct-GGUF](https://huggingface.co/TheBloke/Vigogne-2-7B-Instruct-GGUF) from TheBloke without problem.
There is also a template for [Vigogne instruct](https://github.com/oobabooga/text-generation-webui/blob/main/instruction-templates/Vigogne-Instruct.yaml) in this project (You can change template in : `Parameters` => `Instruction template` of the UI).

From [TheBloke](https://huggingface.co/TheBloke) you can search other versions of `Vigogne` (7B chat, 13B instruct, 13B chat) or others models quantized.
You can also search for other model format (i personally only use GGUF).

I personally don't know another model than [Vigogne](https://github.com/bofenghuang/vigogne) specialy trained for french language.
You can also use others models in french by using french context, but some models doesn't speak French well.

If this don't help you or if you need any informations i will be happy to help you if i can.

PS: Can you furnish more informations about the model you have tried to use or the error you obtain ? (Maybe it can help you or others)
